### DISCUSSION TITLE
how to change the Prompt/Question limit  ?

### DISCUSSION THREAD

**Discussion by xubairp** [mayooear/ai-pdf-chatbot-langchain#7] [Q&A] [answered]
hi, 

 I noticed there is a limit on the number of characters that can be passed in the question/prompt box. how can we increase the limit ?

thanks

**Comment by mayooear** [comment]
Hey can you explain what part of the code you're referring to?

**Comment by xubairp** [comment, marked as answer]
Hi, Please check the attached image, i am referring to the prompt textbox:

![image](https://user-images.githubusercontent.com/128311411/226614731-e1dd8d3b-609f-434a-b618-2a06cbda03f8.png)

**Reply by mayooear** [reply]
In `index.tsx` change `maxLength` number in the `textarea` tag.

```
<textarea
                    disabled={loading}
                    onKeyDown={handleEnter}
                    ref={textAreaRef}
                    autoFocus={false}
                    rows={1}
                    maxLength={512}
                    id="userInput"
                    name="userInput"

```

**Reply by xubairp** [reply]
Thanks
